go/service/tlf.go

```diff
@@ -58,3 +61,45 @@ func (h *tlfHandler) CompleteAndCanonicalizePrivateTlfName(ctx context.Context,
 	ctx = chat.Context(ctx, arg.IdentifyBehavior, &breaks, chat.NewIdentifyNotifier(h.G()))
 	return h.tlfInfoSource.CompleteAndCanonicalizePrivateTlfName(ctx, arg.TlfName)
 }
+
+func (h *tlfHandler) identifyUser(ctx context.Context, assertion string, private bool, idBehavior keybase1.TLFIdentifyBehavior) (keybase1.TLFIdentifyFailure, error) {
```

Comment:
That guy just tests that an identify happens, but yeah the breaks are the key. The semantics of an identify failure is an error is still true on the CLI, but we just don't want the tracker popup from it.